providers/salesloft/metadata/schemas.json

```diff
@@ -218,8 +230,8 @@
         },
         "bulk_jobs_job_data": {
           "displayName": "Job Data for a Bulk Job",
-          "path": "/bulk_jobs_job_data",
```

Comment:
can drop support for this object, since it requires bulk job ID, which we don't have.